### PR TITLE
fix(discord): Video links dont embed

### PR DESCRIPTION
**Pull request by Zickles**
idk this should work lmao

### CODE REVIEW

File: src/views/pages/index.ejs [hunk 1]
```diff
@@ -8,7 +8,11 @@
   <link rel="stylesheet" href="css/index.css">
   <meta name="robots" content="noindex">
   <meta name="twitter:card" content="summary_large_image" />
-  <meta property="og:image" content="<%= img %>" />
+  <% if (img.endsWith('.mp4')) { %>
+    <meta property="og:video" content="<%= img %>" />
```
Comment (Kathund) [inline src/views/pages/index.ejs:12]: <meta name="twitter:player" content="video url">

**Review by Kathund — CHANGES_REQUESTED** [review]

File: src/views/pages/index.ejs (hunk 1 above)
Comment (Kathund) [reply src/views/pages/index.ejs:12]: <meta name="twitter:player" content="Video url">

Comment (Zickles) [reply src/views/pages/index.ejs:12]: you try it

Comment (Zickles) [reply src/views/pages/index.ejs:12]: if it works i will change it

**Pull request closed without merging**